### PR TITLE
added main.js function

### PR DESCRIPTION
**Pull request by Hunter-aw**
worked really hard on this please be gentle

### CODE REVIEW

File: dist/main.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const brancher = () => {
+    console.log(`Wow I'm a branch`)
+}
```
Comment (AlonRR) [inline dist/main.js:3]: blah

File: dist/main.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const brancher = () => {
+    console.log(`Wow I'm a branch`)
```
Comment (corinnegottlieb) [inline dist/main.js:2]: are you really?

Comment (AlonRR) [reply dist/main.js:2]: why so mean

File: dist/main.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const brancher = () => {
```
Comment (Hunter-aw) [inline dist/main.js:1]: yep

File: dist/main.js (hunk 2 above)
Comment (roykesh) [inline dist/main.js:2]: great console.log

**Pull request merged**